### ISSUE TITLE
Error when building

### ISSUE THREAD

**Issue by altano** [DysonSphereMod/QOL#52]
I can 100% reliably cause this crash:

![image](https://user-images.githubusercontent.com/1009/113085215-455f6880-9194-11eb-972f-84cf1394d494.png)

> ArgumentException: An element with the same key already exists in the dictionary.

by pasting this build and clicking it into the world: https://files.terriblefish.com/universal%20matrix%20replication%20-%20layer%201%20-%20equator%20-%20no%20fiber%20optic%20thingers

If I snapshot the source buildings and lay it down again, I generally get the same error within a couple of minutes. I can send my save game if that helps.

**Comment by brokenmass** [comment]
I'm struggling to replicate this error... please send me your savegame (compressed it first) thanks

**Comment by TheIronAngel** [comment]
Hi, I've never had this error before but after updating to the latest version of the mod I've now started getting it occasionally too. Nothing yet I've been able to find that has a 100% repeatability - though it says it has disabled autosave but it literally did one 30 seconds after this error came up.
![2021-04-09_21-34-20](https://user-images.githubusercontent.com/74419136/114160998-a7ca1d00-997b-11eb-92ba-a9d7f00c9622.png)

I have created a save if you want it (150mb before compress) right where it happened, the item being duplicated was one of the 2 INPL Log Stats.
